python_modules/libraries/dagster-dbt/dagster_dbt/include/scaffold/definitions.py.jinja

```diff
@@ -7,6 +7,14 @@ from dagster_dbt import DbtCli, dbt_assets
 dbt_project_dir = Path(__file__).parent.joinpath({{ dbt_project_dir_relative_path_parts | join(', ')}})
 dbt_manifest_path = dbt_project_dir.joinpath("target", "manifest.json")
 
+dbt = DbtCli(project_dir=os.fspath(dbt_project_dir))
+
+if os.getenv("DAGSTER_DBT_BUILD_PROJECT"):
+    dbt_parse_invocation = dbt.cli(["parse"], manifest={})
+    dbt_parse_invocation.wait()
+    dbt_parse_invocation.target_path.joinpath("manifest.json").replace(
```

Comment:
What exactly is this doing / for?

Reply:
We are moving the manifest that's generated in the dynamic target path of a `DbtCli.cli` invocation (e.g. `target-123213`) to the static target path of the dbt project `target/`.

Reply:
Gotcha. That sounds correct, but the dataflow is a little hard to follow, and it overwrites the manifest at the static target path which may or may not be desirable. What if we made something like this possible?

```python
if os.getenv("DAGSTER_DBT_BUILD_PROJECT"):
    dbt_manifest_path = dbt.cli(["parse"]).wait().get_output_manifest_path()
else:
    dbt_manifest_path = dbt_project_dir.joinpath("target", "manifest.json")
```